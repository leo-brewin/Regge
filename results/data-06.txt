# defect
 6.198356078677004930E-02

# i  j   d(defect)/d(lsq(i,j))
  1  2 -1.964668857133379470E+01
  1  3 -1.939660380002190760E+01
  2  3 -1.965447405288413600E+01
  4  1  2.274414261905676980E+01
  4  2  2.299630692039518820E+01
  4  3  2.274156806882957900E+01
  5  1  3.319509757714452910E+00
  5  2  3.360912342322534520E+00
  5  3  3.320048812947515290E+00
  4  5 -2.131336693318224290E+01
  6  1  3.250336097168456140E+00
  6  2  3.387062884847099120E+00
  6  3  3.273941117727506950E+00
  5  6 -2.142289078519109770E+01
  7  1  3.322339423311811280E+00
  7  2  3.359041163046847110E+00
  7  3  3.321754787584243650E+00
  6  7 -2.142182145740466940E+01
  8  1  3.376223869861610920E+00
  8  2  3.288362796384325550E+00
  8  3  3.346348341410931370E+00
  7  8 -2.130859747243189110E+01
  9  1  2.241943050855428940E+01
  9  2  2.281660734171740440E+01
  9  3  2.245188244404216960E+01
  8  9 -2.124646933816129220E+01
  9  4 -3.292771799138696900E+01
