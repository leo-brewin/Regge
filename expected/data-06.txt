# defect
 6.198356078677002720E-02

# i  j   d(defect)/d(lsq(i,j))
  1  2 -1.964668857133379550E+01
  1  3 -1.939660380002190870E+01
  2  3 -1.965447405288413690E+01
  4  1  2.274414261905677140E+01
  4  2  2.299630692039518950E+01
  4  3  2.274156806882958040E+01
  5  1  3.319509757714452990E+00
  5  2  3.360912342322534520E+00
  5  3  3.320048812947515310E+00
  4  5 -2.131336693318224290E+01
  6  1  3.250336097168456120E+00
  6  2  3.387062884847099100E+00
  6  3  3.273941117727506980E+00
  5  6 -2.142289078519109770E+01
  7  1  3.322339423311811230E+00
  7  2  3.359041163046847120E+00
  7  3  3.321754787584243660E+00
  6  7 -2.142182145740466950E+01
  8  1  3.376223869861610870E+00
  8  2  3.288362796384325550E+00
  8  3  3.346348341410931380E+00
  7  8 -2.130859747243189120E+01
  9  1  2.241943050855429020E+01
  9  2  2.281660734171740580E+01
  9  3  2.245188244404217110E+01
  8  9 -2.124646933816129220E+01
  9  4 -3.292771799138697110E+01
